### PR TITLE
Ready to code review

### PR DESCRIPTION
**Pull request by SofiBretz**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Hi @SofiBretz this is a very good start, however, we need to make a few corrections

File: index.css [hunk 1]
```diff
@@ -0,0 +1,330 @@
+.background {
+  background-image: url("https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg");
+  background-repeat: no-repeat;
+  width: 1440px;
```
Comment (Oghenebrume50) [inline index.css]: You do not need to set an explicit width like you did here, please follow up with other places were u set explicit width and make sure they are not affecting the page display

File: index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          <div class="background">
+        <section class="mainWrapper">
+          <h1 class="mainTitle">What will your verse be?</h1>
+          <h2 class="mainDescription row1">Everyone has something to share. See how people are using</h2>
+          <h2 class="mainDescription row2">iPad in their unique ways to add to the world’s story.</h2>
+          <div class="main link">
+          <a href="#">
+            <p id="linkText">Watch the “Your Verse” film and explore the stories <i class="far fa-play-circle"></i></p>
+          </a></div>
+        </section>
```
Comment (Oghenebrume50) [inline index.html]: Edit the section content following this guidelines 
- [ ] There should be three links
- [ ] Each link is an <a> HTML tag, with a <div> tag inside it.
- [ ] The <div> tag  inside the <a> tag should use an image as background, the text on the original page is actually an image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+        <figure class="promos">
+          <img class="promoImg" src="https://web.archive.org/web/20140226015601im_/http://images.apple.com/home/images/promo_30_years_2x.jpg" alt="promoBirthday">
+        </figure>
+
+        <figure class="promos">
+          <img class="promoImg" src="https://web.archive.org/web/20140223180930im_/http://images.apple.com/home/images/promo_supplier_responsibility_2x.jpg" alt="promoSupplier">
+        </figure>
+
+      </section>
```
Comment (Oghenebrume50) [inline index.html]: Move this section tag into the footer and edit it with the same guideline given for the previous section tag.

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Style the page properly add a descriptive readme and submit for re-review.

File: index.html (hunk 3 above)
Comment (SofiBretz) [reply index.html]: Hi! how are you? thank you for your code review :)  I'm not agree that this belongs to the footer :) I think the footer is only the part of the "Shop the Apple Online Store..." Thanks.

**Comment by samgaco** [comment]
### Hi @SofiBretz :hand: 

### Good job so far. Still some points to taking into account before proceeding:

* Try to correct the **failing linters checks.**
* **The four boxes after the main content** should occupy the whole space of the site (horizontally). And each one of them should be a link.

> bottom3-19-15


* **The texts images** in the main content seem to be uncompleted.
* There is too much spacing between these **links in the footer:**

> crop


* And a **suggestion** read about [git flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) so you can apply it or your next projects. 


### Please :pray: submit back for another code review when you are done with these changes :muscle:

**Comment by SofiBretz** [comment]
Dear Samgaco,

For this review: "Try to correct the failing linters checks. The four boxes
after the main content should occupy the whole space of the site
(horizontally) ..." Sorry but this is not a Responsive project. The next
project is Responsive, this one is not. I already arranged your other
recommendations. :) Closing, Can you please be more specific : "The texts
images in the main content seem to be uncompleted." Thank you! :)

Regards,

Sofia Barrantes.

**Comment by samgaco** [comment]
### Hi @SofiBretz :vulcan_salute: 
### I am not asking for you to making the site responsive.

> * The four boxes after the main content should occupy the whole space of the site (horizontally). And each one of them should be a link.

### This is how your project looks like:

tuyaapple

### This is how the original looks like:
originalapple

### I am talking about adapting the sizing of the four boxes to the margins of the site  **not about responsiveness.**

### Regards!

**Comment by samgaco** [comment]
>  Can you please be more specific : "The texts
images in the main content seem to be uncompleted." Thank you! :)

### Seems to be fixed in this update. :+1: 

### They looked like this previously:

> cortados

**Comment by SofiBretz** [comment]
maybe we have different screens because in mine looks ok :) look:
[image: Captura de pantalla (126).png]

**Comment by Oghenebrume50** [comment]
Hi could fix the space at the footer there is too much space between those section elements, asides that everything else is fine, please fix that and submit for re-review

**Comment by SofiBretz** [comment]
Dear Raphael,

I made the correction you requested, this is the maximum that I can manage the
spaces in the footer, because otherwise there is a lot of space between the
left and right text. So the maximum I could do when you look at the
inspector is this. (I attach images) Also remember that in each screen
looks different. I hope this looks better in your screen! ;) Thank you!

Regards,

Sofia.

**Comment by samgaco** [comment]
### Hi @SofiBretz :hand: 

### You've worked hard on this, so congratulations :clap: your project is approved :tada: .

**Adapting our design to different screens and explorers it's a dense topic, and probably could be addressed with responsive design.**

**[OPTIONAL]**  _It is strongly recommended, to solve the linters failing checks before merging to the master branch._

**I hope you keep the good work in your next projects. :muscle:  Regards!**

**Pull request merged**